src/network/instanceMethods.ts

```diff
@@ -19,6 +19,12 @@ async function _post<T, Body, Params = {}>(
   return response.data ?? response;
 }
 
+async function _postAccount<T, Body>(url: string, data: Body): Promise<ApiCallResponse<T>> {
```

Comment:
why this fun name is still _postAccount

Reply:
i restriced user to add any more headers